projects/adrv9001/a10soc/system_qsys.tcl

```diff
@@ -16,9 +16,8 @@ set_instance_parameter_value sys_spi {clockPolarity} {0}
 
 #system ID
 set_instance_parameter_value axi_sysid_0 {ROM_ADDR_BITS} {9}
+set_instance_parameter_value rom_sys_0 {PATH_TO_FILE} "$mem_init_sys_file_path/mem_init_sys.txt"
 set_instance_parameter_value rom_sys_0 {ROM_ADDR_BITS} {9}
 
-set_instance_parameter_value rom_sys_0 {PATH_TO_FILE} "[pwd]/mem_init_sys.txt"
-
 set sys_cstring "sys rom custom string placeholder";
```

Comment:
Shouldn't the `sys_cstring` be removed since this project doesn't have make parameters?